### DISCUSSION TITLE
Creating .torrent

### DISCUSSION THREAD

**Discussion by Aetherinox** [VueTorrent/VueTorrent#1732] [General]
This is going to sound really novice, but I can't seem to figure out if there's a function to actually create `.torrent` files. Obviously I see the ability to add .torrent files to the system, but I don't see any way to generate. Unless I'm just absolutely blind, which would be embarrassing.

Using the docker version. The desktop client has the feature with default theme.

**Edit**: One place I didn't check, was the issues for qb, and I guess I have my answer. Apparently it's surprisingly not even a feature in web yet.
- https://github.com/qbittorrent/qBittorrent/issues/5614

Sorry for the premature post. Just surprised it's not even an offered feature yet for the base client.

**Comment by Larsluph** [comment]
The API has been implemented in qBittorrent but hasn't been released yet.
We're still waiting for that release.

**Reply by Larsluph** [reply]
For any info on the progress of that, checkout the issue that has just opened: #1733

**Comment by Aetherinox** [comment]
Yeah, sorry about that. I got the idea to check the Github issues after my post. Sorta still weird to see that they haven't implemented this yet.

I've been reading up on transmission-cli. I found ctorrent, but that looks like it hasn't been updated in a long time. And then I found some "WebTorrent" app which can create, written in JS / Node.
